backend/app.py

```
@@ -57,37 +63,61 @@ class Config:
+
+@app.get("/api/v1/users/me", response_model=User)
+async def read_users_me(current_user: User = Depends(get_current_active_user)):
+    """Get current user information."""
+    return current_user
 
+# Protected chat endpoints
 @app.post("/api/v1/chat", response_model=ChatMessage)
-async def chat_completion(message: ChatMessage):
+async def chat_completion(
+    message: ChatMessage,
+    current_user: User = Depends(get_current_active_user)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)